nautobot/cloud/templates/cloud/cloudservice_retrieve.html

```diff
@@ -2,7 +2,7 @@
 {% load helpers %}
 
 {% block extra_breadcrumbs %}
-    <li><a href="{% url 'cloud:cloudtype' pk=object.cloud_type.pk %}">{{ object.cloud_type }}</a></li>
+    <li><a href="{% url 'cloud:cloudresourcetype' pk=object.cloud_resource_type.pk %}">{{ object.cloud_resource_type }}</a></li>
 {% endblock extra_breadcrumbs %}
```

Comment:
I didn't change it, but note that `cloudnetwork_retrieve.html` does NOT have a corresponding `extra_breadcrumbs` block. Maybe we should add one?